src/main/java/teammates/common/datatransfer/SqlCourseRoster.java

```diff
@@ -136,23 +138,23 @@ public ParticipantInfo getInfoForIdentifier(String identifier) {
             Student student = getStudentForEmail(identifier);
 
             name = student.getName();
-            teamName = student.getTeamName();
-            sectionName = student.getSectionName();
+            team = student.getTeam();
+            section = student.getSection();
         } else if (isInstructor) {
             Instructor instructor = getInstructorForEmail(identifier);
 
             name = instructor.getName();
-            teamName = Const.USER_TEAM_FOR_INSTRUCTOR;
-            sectionName = Const.DEFAULT_SECTION;
+            team = Const.USER_TEAM_ENTITY_FOR_INSTRUCTOR;
```

Comment:
new should be instantiating the Team here instead of in the Const file. 

eg, `new Team(..., Const.USER_TEAM_FOR_INSTRUCTOR);` 

We never want a shared mutable instance of Team or Section as other classes may modify it and hence it is not constant, leading to hard to trace bugs.